javaScript/package.json

```diff
@@ -4,19 +4,19 @@
     "description": "code-challenges javascript",
     "main": "index.js",
     "scripts": {
-      "test": "jest"
+        "test": "jest"
```

Comment:
I'm unclear exactly what is going on here. Json should have 2 space tabs by convention.